average.go

```
@@ -0,0 +1,26 @@
+package main
+
+import (
+	"fmt"
+	"os"
+)
+
+func average(x []int) float64 {
+	var sum, N int
+	sum = 0
+	N = len(x)
+	if N <= 0 {
+		fmt.Print("Slice has no elements")
+		os.Exit(1)
```

Will be better to return an error in this case. Or just return 0.